UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Common/Logging/EventIds.cs

```diff
@@ -728,7 +728,22 @@ public enum EventIds
         /// <summary>
         /// 809144 - PRODUCT.TXT file and INFO folder delete operation failed
         /// </summary>
-        BessProductTxtAndInfoFolderDeleteFailed = 809144
+        BessProductTxtAndInfoFolderDeleteFailed = 809144,
+
+        /// <summary>
+        /// 809145 - Post product data to PKS started.
```

Comment:
```suggestion
        /// 809145 - Post product data to Product Key Service started.
```

Similarly in other places

Reply:
Resolved